doc-content/enterprise-only/cases/case-management-subcases-con.adoc

```diff
@@ -1,11 +1,40 @@
-[id='case-management-subcases-proc_{context}']
-= Creating sub cases in case management projects
+[id='case-management-subcases-con{context}']
+= Subcases
 
-Subcases give you the flexibility to compose advanced cases that consists of other cases. This means that you can split large and complex cases into multiple layers of abstraction and even multiple case projects. This is similar to splitting a process into multiple subprocesses.
+Subcases give you the flexibility to compose advanced cases that consist of other cases. This means that you can split large and complex cases into multiple layers of abstraction and even multiple case projects. This is similar to splitting a process into multiple subprocesses.
 
 A subcase is another case definition that is invoked from within another case instance or a regular process instance. It has all of the capabilities of a regular case instance:
 
 * It has a dedicated case file.
 * It is isolated from any other case instance.
 * It has its own set of case roles.
 * It has its own case prefix.
+
+You can add subcases to your case definition using the process designer. A subcase is another case within your case project, similar to having a subprocess within your process. Subcases can also be added to a regular business process, which enables you to start a case from within a process instance.
```

Comment:
You can use the process designer to add subcases....
...regular business process. Doing this enables..

Reply:
Done.